sentry/src/main/java/io/sentry/Sentry.java

```diff
@@ -252,12 +272,12 @@ private static synchronized void init(
     // and hub was still NoOp.
     // Registering integrations here make sure that Hub is already created.
     for (final Integration integration : options.getIntegrations()) {
-      integration.register(HubAdapter.getInstance(), options);
+      integration.register(ScopesAdapter.getInstance(), options);
```

Comment:
Could use the `HubScopesWrapper` here to make it compile:
`integration.register(new HubScopesWrapper(ScopesAdapter.getInstance()), options);`

Reply:
done in a follow up pr